beacon_node/execution_layer/src/lib.rs

```diff
@@ -1794,7 +1795,8 @@ impl<T: EthSpec> ExecutionLayer<T> {
                 transactions
                     .into_iter()
                     .map(|tx| {
-                        let mut buf = vec![];
+                        let size = min(tx.size(), T::max_bytes_per_transaction());
```

Comment:
This is a crazy high number, so probably not a good max https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/bellatrix.yaml#L14-L15

Reply:
Oh yea that'd be a gig right? should i just do `tx.size()` ?

Reply:
it's effectively the same max, but even if we bound it to a different max won't we still end up re-allocation until we reach that size? Is there a reason to not do it in one go?

Reply:
due to reverting of the `reth-primitive` related changes this is no longer rethevant